src/main/java/org/kuali/kfs/module/cam/document/service/impl/GlLineServiceImpl.java

```diff
@@ -129,13 +130,18 @@ protected void markCapitalAssetProcessed(GeneralLedgerEntry primary, Integer cap
     protected void deactivateGLEntries(GeneralLedgerEntry entry, Document document, Integer capitalAssetLineNumber) {
         //now deactivate the gl line..
         CapitalAssetInformation capitalAssetInformation = findCapitalAssetInformation(entry.getDocumentNumber(), capitalAssetLineNumber);
-
         if (ObjectUtils.isNotNull(capitalAssetInformation)) {
             List<CapitalAssetAccountsGroupDetails> groupAccountingLines = capitalAssetInformation.getCapitalAssetAccountsGroupDetails();
             Collection<GeneralLedgerEntry> documentGlEntries = findAllGeneralLedgerEntry(entry.getDocumentNumber());
+            if (LOG.isDebugEnabled()) {
+                LOG.debug("deactivateGLEntries, entry.getDocumentNumber(): " + entry.getDocumentNumber() + "  documentGlEntries.size(): " + documentGlEntries.size());
+            }
             for (CapitalAssetAccountsGroupDetails accountingLine : groupAccountingLines) {
                 //find the matching GL entry for this accounting line.
                 Collection<GeneralLedgerEntry> glEntries = findMatchingGeneralLedgerEntries(documentGlEntries, accountingLine);
+                if (LOG.isDebugEnabled()) {
```

Comment:
I think the code would be cleaner and more clear if you remove the debug logging.

Reply:
sure, this can be removed